### PR TITLE
Add system notes to the system extension

### PR DESCRIPTION
**Pull request by linawolf**
Adding main  version for now as the 12.4 only displays the README content until the next release

### CODE REVIEW

**Review by brotkrueml — COMMENTED** [review]
Thought also about it, but opted to wait until the next 12.4 release. This way, we have only one PR, not two.

**Comment by linawolf** [comment]
However now when you announce it ppl will look for it. As main is exactly the same as 12.4 it is equivalent

File: Documentation/Home/SystemExtensions.rst [hunk 1]
```diff
@@ -394,6 +395,27 @@ part of the core and extracted at some point in time.
...
+            Allows to add notes to pages, that can only be seen in the backend.
+
+            `typo3/cms-sys-note <https://packagist.org/packages/typo3/cms-sys-note>`__
+
+         .. container:: card-footer pb-0
+
+            .. rst-class:: horizbuttons-striking-m
+
+            - `main <https://docs.typo3.org/c/typo3/cms-sys-note/main/en-us/>`__
+            - `12.4 <https://docs.typo3.org/c/typo3/cms-seo/12.4/en-us/>`__
```
Comment (brotkrueml) [inline Documentation/Home/SystemExtensions.rst:417]:
```suggestion
            - `12.4 <https://docs.typo3.org/c/typo3/cms-sys-note/12.4/en-us/>`__
```

File: Documentation/Home/SystemExtensions.rst [hunk 2]
```diff
@@ -394,6 +395,27 @@ part of the core and extracted at some point in time.
...
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header h3
+
+            .. rubric:: `System Notes <https://docs.typo3.org/c/typo3/cms-sys-note/main/en-us/>`_
+
+         .. container:: card-body
+
+            Allows to add notes to pages, that can only be seen in the backend.
```
Comment (brotkrueml) [inline Documentation/Home/SystemExtensions.rst:408]: Maybe the text from the introduction is better:
https://docs.typo3.org/c/typo3/cms-sys-note/main/en-us/Editor/Index.html

"The internal notes are a useful feature for adding context and notes to pages. It provides a way for users to document important information related to specific pages."

Comment (linawolf) [inline Documentation/Home/SystemExtensions.rst:408]:
```suggestion
            The internal notes are a useful feature for adding context and notes to pages. 
            It provides a way for users to document important information related to specific pages.
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**